.github/workflows/static-analyze-tests.yml

```diff
@@ -0,0 +1,81 @@
+name: Static Analyze Tests
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'src/adapter/**'
+      - 'src/bridge/**'
+      - 'src/core/**'
+      - 'src/cli/**'
+      - 'tools/**'
+      - 'composer.lock'
+  push:
+    branches: [ 1.x ]
+    paths-ignore:
+      - 'CHANGELOG.md'
+
+# See https://stackoverflow.com/a/72408109
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  static-analyze:
```

Comment:
this can be merged with the other static analysis workflow file, too many workflow files is not great

Reply:
Fixed in https://github.com/flow-php/flow/pull/1329/commits/6f4bf08ddc59b6178e68dc4367086b0c21c4f295